storage/pebble.go

```diff
@@ -153,11 +153,13 @@ func (s *Pebble) GetTxByHash(txHash string) (*types.TxResult, error) {
 	return decodeTx(tx)
 }
 
-func (s *Pebble) BlockIterator(fromBlockNum, toBlockNum uint64) (Iterator[*types.Block], error) {
+func (s *Pebble) BlockIterator(fromBlockNum, toBlockNum uint64, ascending bool) (Iterator[*types.Block], error) {
 	fromKey := keyBlock(fromBlockNum)
 
 	if toBlockNum == 0 {
 		toBlockNum = math.MaxInt64
+	} else {
```

Comment:
What if the `toBlockNum` is `math.MaxUint64`?

Reply:
Good point!
I'll add exception handling for large numbers.